boards/arm/stm32f3_seco_d23/support/openocd.cfg

```diff
@@ -0,0 +1,12 @@
+source [find board/stm32f3discovery.cfg]
```

Comment:
should likely not rely on wrong board script

Reply:
Replacing with actual configuration